src/ORM/DataObjectSchema.php

```diff
@@ -24,6 +24,11 @@ class DataObjectSchema
     use Injectable;
     use Configurable;
 
+    /**
+     * Configuration key for has_one relations that can support multiple reciprocal has_many relations.
+     */
+    public const HASONE_IS_MULTIRECIPROCAL = 'multireciprocal';
```

Comment:
```suggestion
    public const HAS_ONE_MULTI_RELATIONAL = 'multi-relational';
```

`HASONE_IS_MULTIRECIPROCAL` is a pretty weird name IMO

`HAS_ONE_MULTI_RELATIONAL` seems much clearer:
- Uses 'RELATIONAL' instead of 'RECIPROCAL' which lines up with it adding a 'Relation' column. Reciprocal seems much less obvious why it's called that, at least to me
- Removes the redundant "IS"
- Adds in underscores

Seem alright?

Reply:
multi-relation makes it sound like this is multiple relations.... which it isn't. But I'll make this change because I know we'll never agree on this and at the end of the day it's more important that there _is_ a name than that it's a name we agree on.

Reply:
Done

Reply:
It's probably worth noting though that reciprocal [is a word](https://www.merriam-webster.com/dictionary/reciprocal) and was being used correctly, and has a precedent (if a fairly minor one) both [in our docs](https://github.com/search?q=repo%3Asilverstripe%2Fdeveloper-docs%20reciprocal&type=code) and in code (e.g. `DataObject::inferReciprocalComponent()`](https://api.silverstripe.org/5/SilverStripe/ORM/DataObject.html#method_inferReciprocalComponent) (where `component` effectively means relation))